app/views/timeline_entries/edit.html.erb

```diff
@@ -0,0 +1,30 @@
+<%
+  links = [
+    {
+      text: 'Coronavirus pages',
+      href: coronavirus_pages_path
+    },
+    {
+      text: "#{@coronavirus_page.name} timeline entries",
+      href: coronavirus_page_path(slug: @coronavirus_page.slug)
+    },
+    {
+      text: "Edit timeline entry"
+    },
+  ]
+%>
+
+<% content_for :breadcrumbs, render('shared/steps/step_breadcrumb', links: links) %>
+<% content_for :title, formatted_title(@coronavirus_page) %>
+<% content_for :context, "Edit timeline entry" %>
+<% if @timeline_entry.errors.any? %>
+  <%= render "shared/sub_sections/form_errors", resource: @timeline_entry %>
+<% end %>
+
+<div class="covid-edit-page govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with method: :patch, url: coronavirus_page_timeline_entry_path do %>
```

Comment:
`form_for` and `form_tag` have been deprecated and replaced with [`form_with`](https://edgeguides.rubyonrails.org/form_helpers.html), so that's why it's being used here and in the "create" form. Unfortunately though, `form_with` does have its quirks, including using POST as the default form action.

Reply:
@leenagupte so they have! Thanks, i'll go back and update announcements soon to do the same at some point soon.